### PR TITLE
Docs issue 2498

### LINKED ISSUE #2498: create MindsDB TABLES Structure explanation doc.

### PR DESCRIPTION
**Pull request by martyna-mindsdb**
Fixes #2498 

## Please describe what changes you made, in as much detail as possible
  - moved this page out of _SQL API_ section and modified its header
image

  - created intro section
image

  - created `predictors` table section
image
image

  - created `databases` table section
image
image

  - question regarding the `[integration_name]` table section: as I know, integration is an external database connected to MindsDB (in the above screenshot, the `example_db` is an integration), so is this section valid?
image

  - the `model` table section is not completed (is the `model` table a work in progress? because I don't have it in my locally connected MindsDB, nor do I see it in the MindsDB Cloud editor)
image

mindsdb

### CODE REVIEW

**Review by ZoranPandovski — APPROVED** [review]

**Comment by ZoranPandovski** [comment]
Looks great. I have one question, do we want this to be out of the SQL section?

File: docs/mindsdb-docs/mkdocs.yml [hunk 1]
```diff
@@ -90,8 +90,8 @@ nav:
   - Upload a File to MindsDB: sql/create/file.md
   - Train a Predictor: sql/create/predictor.md
   - Get a Prediction: sql/api/select.md
+  - MindsDB Tables Structure: sql/table-structure.md
```
Comment (Ricram2) [inline docs/mindsdb-docs/mkdocs.yml:93]: I would personally put this back into the SQL section.

File: docs/mindsdb-docs/docs/sql/table-structure.md [hunk 2]
```diff
@@ -1,47 +1,134 @@
...
+You can verify it by running the following SQL command:
 
 ```sql
-SHOW TABLES;
+SHOW tables;
+```
+
+On execution, we get:
+
+```
```
Comment (Ricram2) [inline docs/mindsdb-docs/docs/sql/table-structure.md:15]: sql flavour also for outputing tables

File: docs/mindsdb-docs/docs/sql/table-structure.md [hunk 3]
```diff
@@ -1,47 +1,134 @@
...
-+---------------------------+
-| Tables_in_mindsdb         |
-+---------------------------+
-| predictors                |
-| databases                 |
-| integration_name          |
-+---------------------------+
+On execution, we get:
 
+```
```
Comment (Ricram2) [inline docs/mindsdb-docs/docs/sql/table-structure.md:35]: sql flavour

File: docs/mindsdb-docs/docs/sql/table-structure.md [hunk 4]
```diff
@@ -1,47 +1,134 @@
...
-!!! warning "This is a work in progress" 
+```sql
+SELECT *
+FROM mindsdb.predictors;
+```
+
+On execution, we get:
+
+```
++-----------------+--------+------------------+-------+-------------+---------------+-----+-----------------+----------------+
```
Comment (Ricram2) [inline docs/mindsdb-docs/docs/sql/table-structure.md:69]: sql flavour

File: docs/mindsdb-docs/docs/sql/table-structure.md [hunk 5]
```diff
@@ -1,47 +1,134 @@
...
+It stores all the views. [Here](https://docs.mindsdb.com/sql/create/view/) is how you can create a view in MindsDB.
+
+```sql
+SELECT *
+FROM mindsdb.databases;
+```
+
+On execution, we get:
+
+```
```
Comment (Ricram2) [inline docs/mindsdb-docs/docs/sql/table-structure.md:93]: sql flavour

File: docs/mindsdb-docs/docs/sql/table-structure.md [hunk 6]
```diff
@@ -1,47 +1,134 @@
...
+To create a database within MindsDB, follow [this guide](https://docs.mindsdb.com/sql/create/databases/).
+
+```sql
+SELECT *
+FROM mindsdb.databases;
+```
+
+On execution, we get:
+
+```
```
Comment (Ricram2) [inline docs/mindsdb-docs/docs/sql/table-structure.md:123]: sql flavour

**Review by Ricram2 — CHANGES_REQUESTED** [review]

File: docs/mindsdb-docs/mkdocs.yml (hunk 1 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/mkdocs.yml:93]: Done.

File: docs/mindsdb-docs/docs/sql/table-structure.md (hunk 2 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/table-structure.md:15]: Done.

File: docs/mindsdb-docs/docs/sql/table-structure.md (hunk 3 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/table-structure.md:35]: Done.

File: docs/mindsdb-docs/docs/sql/table-structure.md (hunk 4 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/table-structure.md:69]: Done.

File: docs/mindsdb-docs/docs/sql/table-structure.md (hunk 5 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/table-structure.md:93]: Done.

File: docs/mindsdb-docs/docs/sql/table-structure.md (hunk 6 above)
Comment (martyna-mindsdb) [reply docs/mindsdb-docs/docs/sql/table-structure.md:123]: Done.

**Pull request merged**